main/qpdf/APKBUILD

```diff
@@ -1,19 +1,33 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=qpdf
-pkgver=7.0.0
+pkgver=7.1.1
 pkgrel=0
 pkgdesc="Command-line tools and library for transforming PDF files"
-url="http://qpdf.sourceforge.net/"
+url="http://qpdf.sourceforge.net"
 arch="all"
-license="Artistic-2.0"
+license="Apache-2.0"
 depends=""
-# coreutils needed for tests
-makedepends="zlib-dev libjpeg-turbo-dev bash coreutils perl"
+makedepends="zlib-dev libjpeg-turbo-dev bash perl"
+checkdepends="coreutils"
 install=""
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-fix-qdf:fix_qdf:noarch"
-source="http://downloads.sourceforge.net/sourceforge/qpdf/qpdf-$pkgver.tar.gz"
+subpackages="$pkgname-dev
+	$pkgname-doc
+	$pkgname-libs
+	$pkgname-fix-qdf:fix_qdf:noarch
+	"
+source="http://downloads.sourceforge.net/sourceforge/$pkgname/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
 
-builddir="$srcdir"/qpdf-$pkgver
+# secfixes:
```

Comment:
Could you add the secfixes comment in a seperate commit with the commit message `main/qpdf: add secfixes comment`?

Reply:
Added secfixes to a separate commit.

Reply:
Yes. Added comments to a separate commit.